Bukkit/src/main/java/com/plotsquared/bukkit/queue/BukkitChunkCoordinator.java

```diff
@@ -197,9 +205,11 @@ private void requestBatch() {
         BlockVector2 chunk;
         for (int i = 0; i < this.batchSize && (chunk = this.requestedChunks.poll()) != null; i++) {
             // This required PaperLib to be bumped to version 1.0.4 to mark the request as urgent
+            loadingChunks++;
             PaperLib
                     .getChunkAtAsync(this.bukkitWorld, chunk.getX(), chunk.getZ(), true, true)
                     .whenComplete((chunkObject, throwable) -> {
+                        loadingChunks--;
```

Comment:
If I see this correctly, `loadingChunks` is accessed from different threads? (looking at the code below)

Reply:
Yeah that's what I'd expect, but then the code below should probably removed at some point